docs/multipass.md

```diff
@@ -0,0 +1,21 @@
+# Multipass
+[Shopify Multipass - Shopify Documentation](https://shopify.dev/docs/admin-api/rest/reference/plus/multipass) <br>
+This helper class generates token or URL that's needed for Shopify Multipass login (This feature is only available for Plus Store)
+```python
+from spylib import Multipass
+
+
+# Customer email is required to generate token or URL
+customer_data = {"email": "customer@email.com"}
+
+# Initiate the Multipass class
+multipass = Multipass("MULTIPASS_SECRET")
+
+# Generate URL
+url = multipass.generate_url(customer_data, "https://{ShopifyDomain}.myshopify.com")
+# https://{ShopifyDomain}.myshopify.com/account/login/multipass/{MultipassToken}
+
+# Generate token
+token = multipass.generate_token(customer_data)
```

Comment:
just curious, in what occasion would the library user would want to return the token? I am thinking if the `generate_token` is kinda like a private / internal method that the library user doesn't need to know/use.

Reply:
Since it's done and it doesn't hurt to expose it, let's keep it that way. Maybe we can't think of a scenario but it exists nonetheless.

Reply:
@ponty33 Maybe let's have a sentence "If for some reason you need it, you can also generate separately the token used in the URL:" just above the `generate_token` example in the docs.